src/main/java/org/xbill/DNS/lookup/LookupSession.java

```diff
@@ -0,0 +1,314 @@
+// SPDX-License-Identifier: BSD-2-Clause
+package org.xbill.DNS.lookup;
+
+import static java.lang.String.format;
+import static java.util.Collections.singletonList;
+import static java.util.concurrent.CompletableFuture.*;
```

Comment:
Wildcard imports are not allowed in g-j-f. Seems I forgot to bind `checkstyle:check` to the verify phase (and I'd still prefer not using static method imports at all).

Reply:
Moved to spelling out the classes instead